ResourceStatusSystem/Command/FreeDiskSpaceCommand.py

```diff
@@ -0,0 +1,102 @@
+''' FreeDiskSpaceCommand
+
+  The Command gets the free space that is left in a DIRAC Storage Element
+
+'''
+
+from datetime                                                   import datetime
+from DIRAC                                                      import S_OK, S_ERROR, gConfig
+from DIRAC.ResourceStatusSystem.Command.Command                 import Command
+from DIRAC.Core.DISET.RPCClient                                 import RPCClient
+
+__RCSID__ = '$Id:  $'
+
+
+class FreeDiskSpaceCommand( Command ):
+  '''
+  Uses diskSpace method to get the free space
+  '''
+
+  def __init__( self, args = None ):
+
+    super( FreeDiskSpaceCommand, self ).__init__( args )
+
+    self.rpc = None
+    self.rsClient = None
+
+  def getUrl(self, SE, protocol = None ):
```

Comment:
Seems like my previous comment was lost, but please don't use directly gConfig for that, use the StorageFactory, and don't construct the URLs yourself but use the getBaseURL method of the SE plugins